Source/Libraries/GSF.Core/Data/DataDeleter.cs

```diff
@@ -232,7 +232,9 @@ private void ExecuteDeletes(Table fromTable, Table toTable)
             OnOverallProgress((int)m_overallProgress, (int)m_overallTotal);
 
             // Execute source query
-            using (IDataReader fromReader = fromTable.Connection.ExecuteReader("SELECT " + fieldCollection.GetList(sqlEscapeFunction: fromTable.Parent.Parent.SQLEscapeName) + " FROM " + fromTable.SQLEscapedName, CommandBehavior.SequentialAccess, Timeout))
+            using (IDataReader fromReader = fromTable.Connection.ExecuteReader("SELECT {0} FROM {1}",
```

Comment:
Parametersubstition does not work on Fielnames or Tablenames

Reply:
updated